etc/config/c.amazon.properties

```diff
@@ -2605,6 +2605,11 @@ compiler.movfuscatortrunk.exe=/opt/compiler-explorer/movfuscator-trunk/build/lcc
 compiler.movfuscatortrunk.semver=(trunk)
 compiler.movfuscatortrunk.name=movfuscator (trunk)
 
+#################################
+# LC3 compiler
+compiler.lc3.name=LC3
+compiler.lc3.exe=/opt/compiler-explorer/C-LC3-Compiler/build/lc3-compile
```

Comment:
Which version do you want installed? what is `"build` ? is this a trunk build? or a version? I need to know - we can build a particular version or a daily turnk build - up to you!

Reply:
we spoke on Discord; will do a trunk build.